apps/music/test/marionette/Player_test.js

```diff
@@ -133,4 +133,39 @@ marionette('Music player tests', function() {
       music.checkPlayerIconShown(true);
     });
   });
+
+  suite('Rating test', function() {
+    test('Rating persistance', function() {
+      music.launch();
+      music.waitForFirstTile();
+      music.switchToSongsView();
```

Comment:
You could add a check here to verify that the song is unrated and no starts are on at this point.